File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n=nums.length;
        int [] output= new int[n*2];
        for (int i=0; i<2; i++)
        {
            for (int j=0;j<n; j++)
            {
                output[i*n+j]=nums[j];
            }
        }
        return output;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if not nums:
            return 0
        max_count=1
        count=1
        s=sorted(set(nums))
        for i in range(1,len(s)):
            if s[i]==s[i-1]+1:
                count+=1
                max_count=max(count,max_count)
            else:
                count=1
        return max_count      